frameworks/bitcoin-move/sources/bitseed.move

```diff
@@ -180,4 +213,45 @@ module bitcoin_move::bitseed {
         seed: vector<u8>, seed_key: vector<u8>,
         user_input: vector<u8>, user_input_key: vector<u8>,
     ): vector<u8>;
-}
\ No newline at end of file
+
+
+    // ==== Process Bitseed ==== //
+    public fun process(tx: &Transaction) {
+        let txid = types::tx_id(tx);
+        let txoutput = types::tx_output(tx);
+        let idx = 0;
+        let txoutput_len = vector::length(txoutput);
+        while(idx < txoutput_len){
+            let vout = (idx as u32);
+            let output_point = types::new_outpoint(txid, vout);
+            let utxo_obj = utxo::borrow_utxo(output_point);
+            let utxo = object::borrow(utxo_obj);
+            let seals = utxo::get_seals<Inscription>(utxo);
+
+            // Track the Inscription via SatPoint
+            let j = 0;
+            let seals_len = vector::length<ObjectID>(&seals);
+            while(j < seals_len){
+                let seal_object_id = *vector::borrow(&seals, j);
+                let inscription_obj = object::borrow_object<Inscription>(seal_object_id);
+                let inscription = object::borrow(inscription_obj);
+                process_inscription(tx, inscription);
+
+                j = j + 1;
+            };
+
+            idx = idx + 1;
+        };
+    }
+
+    public fun process_inscription(tx: &Transaction, inscription: &Inscription) {
+        let txid = types::tx_id(tx);
+        let index = ord::index(inscription);
+        let inscription_id = ord::new_inscription_id(txid, index);
+
+        if (is_bitseed(inscription)) {
+            // TODO parse inscription.meta and valid op
```

Comment:
Since the meta of Inscription is cbor encoded, a decoding function is now needed.